enaml/qt/qt_button_group.py

```diff
@@ -0,0 +1,28 @@
+from atom.api import Typed
+from enaml.widgets.button_group import ProxyButtonGroup
+
+from .QtWidgets import QButtonGroup
+
+from .qt_toolkit_object import QtToolkitObject
+
+
+class QtButtonGroup(QtToolkitObject, ProxyButtonGroup):
+    widget = Typed(QButtonGroup)
```

Comment:
Same here a docstring for the class and the attribute (feel free to copy paste from other classes for example the following is often used for widget #: A reference to the widget created by the proxy.)